templates/definition/charger/ocpp.yaml

```diff
@@ -1,25 +1,46 @@
 template: ocpp
 products:
   - description:
-      de: OCPP v1.6 kompatible Wallbox mit Smart Charging Profile
-      en: OCPP v1.6 compatible charger with smart charging profile
+      de: OCPP v1.6 Websocket/JSON kompatible Wallbox mit Smart Charging Profile
+      en: OCPP v1.6 Websocket/JSON compatible charger with smart charging profile
 group: generic
 requirements:
   description:
     de: |
+      Bei OCPP verbindet sich die Wallbox (Client) zu evcc (Server).
+      Die Wallbox muss daher evcc via Hostname (funktionierende DNS-Auflösung erforderlich!) oder über die IP-Adresse auf Port 8887 erreichen können.
+      Standardmäßig wird die erste eingehende Verbindung mit einer beliebigen Ladepunkt-Kennung verwendet.
+      Um mehrere Ladepunkte eindeutig zuordnen zu können müssen die jeweilige Stationskennung (`stationid: `) und Anschlussnummer (`connector: `) hinterlegt werden.
+      Gegebenenfalls benötigt der Ladepunkt eine vorkonfigurierte (virtuelle) Token-ID/RFID-Kennung (`idtag: `) mit der die Ladevorgänge ohne Authentifizierung gestartet werden können.
+      Wenn die Wallbox das Senden von Zählerdaten unterstützt sollte dafür, wenn möglich, ein kurzes Zeitintervall gewählt werden.
+
       Voraussetzungen:
-      * Löschen zuvor konfigurierter Ladeprofile (z.B. durch ein anderes Backend)
-      * Konfiguration des Backends der Wallbox: `ws://evcc.local:8887/`
+      * Ggf. zuvor konfigurierte OCPP-Profile (z.B. durch eine andere Backend-Anbindung) in der Wallbox-Konfiguration entfernen
+      * Backend-URL (Central System) in der Wallbox-Konfiguration: `ws://evcc.local:8887/` bzw. `ws://192.0.2.2:8887/`
+      * Protokoll: OCPP-J v1.6, ocpp16j, JSON, Websocket, ws:// o.ä.
+      * Keine Verschlüsselung, keine Authentifizierung, kein Passwort
+      * LAN-Verbindung
```

Comment:
Ist keine Voraussetzung

Reply:
Wie würde man alternativ per Mobilfunk ins lokale Netz kommen?

Reply:
Das mir eigentlich egal wie der Anwender das macht. Tatsächlich würde es ja sogar mit einer remoten Box laufen!

Reply:
In der Tat. Ich hab nur etwas Angst vor den ganzen Anfragen von Einsteigern die vielleicht nicht verstehen dass dafür noch etwas andere Voraussetzungen geschaffen werden müssen. Ähnlich wie bei Docker & Co.

Reply:
Dann lass uns eher sagen "im lokalen Netzwerk" oder sowas?

Reply:
So?
```suggestion
      * Verbindung über das lokale Netzwerk
```